go.mod

```diff
@@ -14,13 +14,14 @@ require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/binance-chain/tss-lib v1.3.1
 	github.com/btcsuite/btcd v0.20.1-beta
+	github.com/celo-org/celo-blockchain v0.0.0-20210222234634-f8c8f6744526
 	github.com/ethereum/go-ethereum v1.9.10
 	github.com/gogo/protobuf v1.3.1
 	github.com/google/gofuzz v1.1.0
 	github.com/ipfs/go-log v1.0.4
-	github.com/keep-network/keep-common v1.3.1-0.20210217111137-6949bf58ebd8
-	github.com/keep-network/keep-core v1.3.2-0.20210217115335-806bf9a58b01
-	github.com/keep-network/tbtc v1.1.1-0.20210217115559-4a6a60e5274a
+	github.com/keep-network/keep-common v1.3.1-0.20210224153316-e59366b9ba03
```

Comment:
We can take the opportunity and update the dependency to include the change from https://github.com/keep-network/keep-common/pull/68.

Reply:
See https://github.com/keep-network/keep-ecdsa/pull/686/commits/3e2b29eb7a83e8cb3b8e38c0dd3cdad6f65cba4d. It bumps `keep-common` to the head of https://github.com/keep-network/keep-common/pull/69 which also embeds changes from https://github.com/keep-network/keep-common/pull/68